### ISSUE TITLE
[1.15.2] Bugs with vertical slabs + UV Lock suggestion

### ISSUE THREAD

**Issue by maxim0p** [CJMinecraft01/DoubleSlabs#44]
**1)** There is a lighting bug that occurs typically when there are full blocks next to or on a full block , this is only visible with smooth lighting turned on of course.
![](https://i.imgur.com/iJsqNz6.png)

![](https://i.imgur.com/ItTZ3Cg.png)

**2)** One side of a double vertical slab becomes invisible when one of the sides is covered by a block
https://i.imgur.com/1ygcaoQ.mp4

I would also like to suggest UV locking the vertical slabs to blend into builds better, and to mimic how other vertical slab mods look. While I think having rotated textures are nice in some instances, I think generally having consistent uniform textures would look better in builds.
![](https://i.imgur.com/BaWdHcy.png)

The left is a wall with vertical stone brick slabs, the right is a wall with just regular stone bricks.
Basically i'm proposing to make the vertical slabs textures on the left look how full regular blocks look on the right. I hope this is implementable, although I have no idea how you accomplished this magic, but assuming it uses blockstates you can turn UV lock on with a tag. https://minecraft.gamepedia.com/Model#Block_states

**Comment by CJMinecraft01** [comment]
Thanks for these suggestions and bug reports! These should be fixed in the next version. If there are any further graphical issues please let me know! And just to let you know that this doesn't use blockstates but in fact rotates the original half slab model retaining all of the properties of the original model. To fix this, I'm just rotating it again so they all align in the same direction.

**Comment by CJMinecraft01** [comment]
I've tweaked how this works and so it should be fixed in the latest version!
